integration/external/Services.e2e.test.ts

```diff
@@ -264,6 +298,12 @@ describe('Gate-keeping of Web Services using NFT ERC-721 End-to-End', () => {
     })
 
     it('The subscriber can access the open service endpoints through the proxy', async () => {
+      if (SKIP_OPEN_ENDPOINT) {
+        console.log(`Skipping Open Endpoints test because SKIP_OPEN_ENDPOINT is set to true`)
+        assert.isTrue(true)
+        return
+      }
+
```

Comment:
```suggestion
    it('The subscriber can access the open service endpoints through the proxy', async function () {
      if (SKIP_OPEN_ENDPOINT) {
        console.log(`Skipping Open Endpoints test because SKIP_OPEN_ENDPOINT is set to true`)
        this.skip()
      }

```